Mage.Tests/src/test/java/org/mage/test/cards/copy/SparkDoubleTest.java

```diff
@@ -68,7 +68,6 @@ public void test_CopyPlaneswalkerWithoutLegendaryWithOneCounter() {
         castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Spark Double");
         setChoice(playerA, true);
         setChoice(playerA, "Ajani, the Greathearted");
-        setChoice(playerA, "Ajani, the Greathearted"); // two etb effects (own + copy)
```

Comment:
Must be two effects for choose. Why it disappear? User cant choose a starting loyalty effect now?

Reply:
It's not applying the way it used to, even though it's technically still an etb effect. It shouldn't matter as it's only adding counters.